From 0aa88ab599c36f0981c78476f76add50649f5b59 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:50 -0400
Subject: [redhat] kabi: Adding symbol call_usermodehelper_setup (kernel/kmod.c)

Message-id: <b0bde36c3bec963bd44d99e7e5037405b08c4602.1410510773.git.poros@redhat.com>
Patchwork-id: 95975
O-Subject: [PATCH RHEL7.1 050/233] kabi: Adding symbol call_usermodehelper_setup (kernel/kmod.c)
Bugzilla: 1071097
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Prepare to call a usermode helper.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/call_usermodehelper_setup
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/call_usermodehelper_setup
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/call_usermodehelper_setup

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/call_usermodehelper_setup b/redhat/kabi/kabi-module/kabi_ppc64/call_usermodehelper_setup
new file mode 100644
index 0000000..ff7af4f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/call_usermodehelper_setup
@@ -0,0 +1,2 @@
+#1-
+0x87236b53 call_usermodehelper_setup vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/call_usermodehelper_setup b/redhat/kabi/kabi-module/kabi_s390x/call_usermodehelper_setup
new file mode 100644
index 0000000..977825d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/call_usermodehelper_setup
@@ -0,0 +1,2 @@
+#1-
+0x116b66a7 call_usermodehelper_setup vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/call_usermodehelper_setup b/redhat/kabi/kabi-module/kabi_x86_64/call_usermodehelper_setup
new file mode 100644
index 0000000..10328cd
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/call_usermodehelper_setup
@@ -0,0 +1,2 @@
+#1-
+0xe63031e1 call_usermodehelper_setup vmlinux EXPORT_SYMBOL
-- 
1.7.1